src/EnergyPlus/CMakeLists.txt

```diff
@@ -1147,6 +1147,17 @@ if(BUILD_TESTING)
       COMMAND  energyplus -D -d "${CLI_TEST_DIR}/PythonPlugin.FromOutside/out-${NON_ASCII_DIRNAME}" ${NON_ASCII_DIRNAME}/${TEST_CASE}.idf
       WORKING_DIRECTORY  "${CLI_TEST_DIR}"
     )
+
+    set(TEST_DIR "${PROJECT_BINARY_DIR}/tst/api/TestRuntimeReleasesTheGIL")
+    file(MAKE_DIRECTORY ${TEST_DIR})
+    add_test(NAME "API.Runtime.PythonPlugin.TestRuntimeReleasesTheGIL"
+      COMMAND "${Python_EXECUTABLE}" "${PROJECT_SOURCE_DIR}/tst/EnergyPlus/api/TestRuntimeReleasesTheGIL.py" -d "${TEST_DIR}" -w "${EPW_FILE}" -D "${PROJECT_SOURCE_DIR}/tst/EnergyPlus/api/TestRuntimeReleasesTheGIL/mcve_gil.idf"
+    )
+    set_tests_properties("API.Runtime.PythonPlugin.TestRuntimeReleasesTheGIL"
+      PROPERTIES
+        ENVIRONMENT PYTHONPATH=${DIR_WITH_PY_ENERGYPLUS}
+        TIMEOUT 10  # This used to timeout! and we expect it NOT to
+    )
```

Comment:
Register the ctest. **NOTE THE TIMEOUT 10 (seconds)**, we do expect it to timeout to demonstrate #10842